packages/application-components/src/components/custom-views/custom-panel/custom-panel.tsx

```diff
@@ -0,0 +1,31 @@
+import { ReactNode } from 'react';
+import styled from '@emotion/styled';
+import { designTokens } from '@commercetools-uikit/design-system';
+import ModalPage from '../../modal-pages/internals/modal-page';
+
+type TCustomPanelProps = {
+  title: string;
+  size: 'small' | 'large';
+  onClose: () => void;
+  children: ReactNode;
+};
+
+const ContentWrapper = styled.div`
+  padding: ${designTokens.spacing40} 40px;
+`;
+
+function CustomPanel(props: TCustomPanelProps) {
+  return (
+    <ModalPage
+      hidePathLabel
+      isOpen
```

Comment:
Just checking if it's expected the modal to always being open on render

Reply:
Yes it is.